ugs-platform/ugs-platform-ugscore/src/main/java/com/willwinder/ugs/nbp/core/actions/PortAction.java

```diff
@@ -132,9 +114,34 @@ else if (evt.isStateChangeEvent()) {
             c.setVisible(!backend.isConnected());
         }
     }
-    
+
     @Override
     public Component getToolbarPresenter() {
+        if (c == null) {
```

Comment:
Does this change make startup faster? Should this function be synchronized in case it gets called twice?

Reply:
There is only one toolbar allowed at a time so I think that this is safe.